src/templates/javascript.ts

```diff
@@ -0,0 +1,129 @@
+import type { Template } from '../lib/shared';
+import { Directory } from '../lib/tree/Directory';
+
+export const template: Template = {
+	main: 'src/{name}.js',
+	scripts: {
+		lint: 'eslint src --ext js,mjs --fix',
+		format: 'prettier --write {src}/**/*.js',
+		prepare: 'husky install .github/husky'
+	},
+	dependencies: ['discord.js', '@sapphire/framework', '@sapphire/plugin-logger'],
+	devDependencies: [
+		'@commitlint/cli',
+		'@commitlint/config-conventional',
+		'@sapphire/eslint-config',
+		'@sapphire/prettier-config',
+		'prettier',
+		'husky',
+		'lint-staged'
```

Comment:
```suggestion
		'@sapphire/eslint-config',
		'@sapphire/prettier-config',
		'@sapphire/ts-config'
```

See other comment about hooks and such. And we can remove prettier because it is a dependency of the 2 configs.